android/src/main/java/expo/modules/xmtpreactnativesdk/wrappers/ContentJson.kt

```diff
@@ -41,12 +43,13 @@ class ContentJson(
             Client.register(TextCodec())
             Client.register(AttachmentCodec())
             Client.register(ReactionCodec())
+            Client.register(RemoteAttachmentCodec())
             Client.register(ReplyCodec())
             // TODO:
             //Client.register(CompositeCodec())
             //Client.register(GroupChatMemberAddedCodec())
             //Client.register(GroupChatTitleChangedCodec())
```

Comment:
I removed these from the sdks since we pivoted on the group chat approach so we should remove them from here.